scout/adapter/mongo/case.py

```diff
@@ -506,7 +506,11 @@ def load_case(self, config_data, update=False):
             LOG.warning(error)
 
         if existing_case and update:
-            self.update_case(case_obj)
+            case_obj['rerun_requested'] = False
+            if case_obj['status'] in ['active', 'archived']:
+                case_obj['status'] = 'inactive'
+
+            self.update_case(case_obj, keep_date=False)
```

Comment:
You don't need to pass the keep_date here really, but maybe it's more clear this way?

Reply:
:blush: yes, I know - I just kept loosing track of which calls I had edited when it wasn't there..

Reply:
ah ok